.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: Run watir-rails CI
+
+'on':
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.ruby-version == 'debug' }}
+    strategy:
+      matrix:
+        BUNDLE_GEMFILE: [gemfiles/Gemfile.rails-3.x, gemfiles/Gemfile.rails-4.x, Gemfile]
+        ruby-version: [2.6.3, debug]
+        exclude:
+          # rails-4.2 requires BigDecimal.new (dropped in ruby-2.7)
+          - { BUNDLE_GEMFILE: gemfiles/Gemfile.rails-4.x, ruby-version: debug }
+          # The issue on rails-3.2 + head is json-1.8.6 compat (simplecov
+          # uses it), we could potentially workaround it using, for example, Oj
+          - { BUNDLE_GEMFILE: gemfiles/Gemfile.rails-3.x, ruby-version: debug }
+    env:
+      BUNDLE_GEMFILE: ${{ github.workspace }}/${{ matrix.BUNDLE_GEMFILE }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby-version }}
+          # rails < 5 doesn't support bundler-2
+          bundler: 1
+          bundler-cache: true
+
+      - run: bundle exec rake
+
+      - uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Does this mean that `watir-rails` repo needs to configure this secret after PR has been merged so that coveralls would work properly?

Reply:
`GITHUB_TOKEN` is a special one, it's always set (see [here](https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret)) and `coverallsapp/github-action@master` is getting repo and other stuff from other default envvars, so it works out-of-the box.